packages/setuptools.rb

```diff
@@ -3,33 +3,23 @@
 class Setuptools < Package
   description 'Easily download, build, install, upgrade, and uninstall Python packages'
   homepage 'https://pypi.org/project/setuptools/'
-  version '46.1.3'
+  version '50.3.2'
   compatibility 'all'
-  source_url 'https://github.com/pypa/setuptools/archive/v46.1.3.tar.gz'
-  source_sha256 '984eeb7909fa610a441711c00dc4fa5bbf3c49f9b4b5204b56b7362bdd04f397'
+  source_url 'https://github.com/pypa/setuptools/archive/v50.3.2.tar.gz'
+  source_sha256 '7d97c001ce9193c6d947bc584b6a19f593e1d2dd4d6c443de3b1e545875bc132'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/setuptools-46.1.3-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/setuptools-46.1.3-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/setuptools-46.1.3-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/setuptools-46.1.3-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '480fa13c1f2ef97d31e08ec9da385f71034a878ec6a19ccfa7ef4fc776a12142',
-     armv7l: '480fa13c1f2ef97d31e08ec9da385f71034a878ec6a19ccfa7ef4fc776a12142',
-       i686: '758a37bddd867c069f237a6d70e56377e088391f1a75e1d81a230838aa5cd242',
-     x86_64: '58f237baeb6e5de572d63a357d8b5c92288112db572f619bb40713319692a94c',
-  })
 
-  depends_on 'python27'
   depends_on 'python3'
 
   def self.build
     system 'python3 bootstrap.py'
   end
 
   def self.install
-    system "python2 setup.py install --root #{CREW_DEST_DIR} --prefix #{CREW_PREFIX}"
+    # Install pip for python2 first using last supported systemtools version (<=v45) for python27.
+    system "curl -Lf https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python2 ./get-pip.py setuptools==44.1.1 --root=#{CREW_DEST_DIR} --prefix=#{CREW_PREFIX} "
```

Comment:
Would this work also?
```
system "pip2 install setuptools==44.1.1 --root=#{CREW_DEST_DIR} --prefix=#{CREW_PREFIX}"
```

Reply:
Maybe? But fyi the pip installed seems to be an older version in python27 too:
```
chronos@mm86x86_64 ~ $ pip2 -v list
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Package    Version             Location                               Installer
---------- ------------------- -------------------------------------- ---------
pip        19.2.3              /usr/local/lib/python2.7/site-packages pip
setuptools 46.1.3.post20200326 /usr/local/lib/python2.7/site-packages
WARNING: You are using pip version 19.2.3, however version 20.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
chronos@mm86x86_64 ~ $ pip2 install setuptools==44.1.1
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Collecting setuptools==44.1.1
  Downloading https://files.pythonhosted.org/packages/e1/b7/182161210a13158cd3ccc41ee19aadef54496b74f2817cc147006ec932b4/setuptools-44.1.1-py2.py3-none-any.whl (583kB)
     |################################| 583kB 4.1MB/s
Installing collected packages: setuptools
  Found existing installation: setuptools 46.1.3.post20200326
    Uninstalling setuptools-46.1.3.post20200326:
      Successfully uninstalled setuptools-46.1.3.post20200326
Successfully installed setuptools-44.1.1
WARNING: You are using pip version 19.2.3, however version 20.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```
In contrast with python3:
```
chronos@mm86x86_64 ~ $ pip3 -v list
Package    Version Location                               Installer
---------- ------- -------------------------------------- ---------
pip        20.3.1  /usr/local/lib/python3.9/site-packages pip
setuptools 49.2.1  /usr/local/lib/python3.9/site-packages pip

Reply:
Also, maybe this?
```pip2 install --upgrade pip```